Update issue template (#2452)
splunk/splunk-connect-for-syslog: .github/ISSUE_TEMPLATE/bug-escalation-report.md

**What is the sc4s version ?**

> **What OS (with version) using for SC4S hosting?**

Reviewer: ```suggestion
**Which operating system (including its version) are you using for hosting SC4S?**
```

> **What the runtime (docker / podman / docker swarm / byoe / microk8s) using for SC4S?**

Reviewer: ```suggestion
**Which runtime (Docker, Podman, Docker Swarm, BYOE, MicroK8s) are you using for SC4S?**
```
